app/workers/subject_metadata_worker.rb

```diff
@@ -0,0 +1,20 @@
+class SubjectMetadataWorker
+  include Sidekiq::Worker
+
+  def perform(subject_set_id)
+    update_sms_priority_sql = <<-SQL
+      UPDATE set_member_subjects
+      SET    priority = CAST(subjects.metadata->>'#priority' AS INTEGER)
```

Comment:
```suggestion
      SET    priority = CAST(subjects.metadata->>'#priority' AS NUMERIC)
```

Ensure we preserve the mantissa for folks that pass in decimal numbers